### DISCUSSION TITLE
Package `Microsoft.AspNetCore.Authentication` deprecated.

### DISCUSSION THREAD

**Discussion by ricardoo-cavalheiro** [balta-io/forum#381] [Tira Dúvidas] [answered]
Dentro do curso `Fundamentos do ASP.NET 6` na parte de `Autenticação e Autorização` o pacote `Microsoft.AspNetCore.Authentication` está marcado como depreciado. Estou apenas documentando. Apesar de não recomendado, acredito que dá pra continuar usando esse pacote só pra finalizar o curso.

**Comment by andrebaltieri** [comment, marked as answer]
Cobriremos isto nos novos cursos @ricardo-passos! Obrigado pelo feedback
